src/context/fetcher/TenantBillingDetails.tsx

```diff
@@ -66,17 +66,28 @@ export const useTenantBillingDetails = () => {
     return useContext(TenantContext);
 };
 
-export const useTenantUsesExternalPayment = (selectedTenant: string) => {
+export const useTenantUsesExternalPayment = (
+    selectedTenant: string
+): [boolean, string | null] => {
```

Comment:
Should the string type be narrowed to the set of supported providers since its value is, ultimately, determined by the client?

Reply:
It will eventually for sure